From b62048701cea5024383e19314592f2edcd9810fd Mon Sep 17 00:00:00 2001
From: Stephan Bergmann <sbergman@redhat.com>
Date: Sat, 16 Feb 2013 21:39:13 +0000
Subject: Work around problem with boost::shared_array(NULL) ctor and Boost 1.53.0

...claiming

  template<class Y> boost::shared_array<T>::shared_array(Y*)

is not a viable option due to mismatched types 'Y*' and 'long int'

Change-Id: I8db321cd25cd73c84fa2a3124c9ec1018c131d5f
---
diff --git a/vcl/source/gdi/svgdata.cxx b/vcl/source/gdi/svgdata.cxx
index 01707e6..e64646f 100644
--- a/vcl/source/gdi/svgdata.cxx
+++ b/vcl/source/gdi/svgdata.cxx
@@ -166,7 +166,7 @@ SvgData::SvgData(const SvgDataArray& rSvgDataArray, sal_uInt32 nSvgDataArrayLeng
 
 //////////////////////////////////////////////////////////////////////////////
 SvgData::SvgData(const OUString& rPath):
-    maSvgDataArray(NULL),
+    maSvgDataArray(),
     mnSvgDataArrayLength(0),
     maPath(rPath),
     maRange(),
--
cgit v0.9.0.2-2-gbebe
